### DISCUSSION TITLE
patternProperties 和 additionalProperties 这些在什么场景里能用到？

### DISCUSSION THREAD

**Discussion by chenos** [alibaba/formily#2699] [Q&A 2.x] [answered]
http://json-schema.org/understanding-json-schema/reference/object.html#pattern-properties

在 JSON Schema 里能理解是干什么的，但是在 Formily 的 Schema 里有什么作用呢？

**Comment by janryWang** [comment, marked as answer]
目前暂时没啥用，保留属性
